nbs/examples/Large_Datasets_Tutorial.ipynb

```diff
@@ -0,0 +1,726 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    nf = NeuralForecast(models=models, freq='M')
</pre><p>Please use <code>freq='ME'</code> here to address possible deprecation warnings from pandas</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1074/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>